hadoop-hdds/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockInputStream.java

```diff
@@ -365,8 +365,7 @@ protected synchronized int readWithStrategy(ByteReaderStrategy strategy)
     int totalRead = 0;
     while (strategy.getTargetLength() > 0 && remaining() > 0) {
       int currentIndex = currentStreamIndex();
-      try {
-        BlockExtendedInputStream stream = getOrOpenStream(currentIndex);
+      try (BlockExtendedInputStream stream = getOrOpenStream(currentIndex)) {
```

Comment:
We want to keep these sub-streams open.  They should be closed when the `ECBlockInputStream` is closed:

https://github.com/apache/ozone/blob/516bc9659b2d4d2567c7570e43d0eb012b666f5a/hadoop-hdds/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockInputStream.java#L454-L456